### PR TITLE
[RU] Translation update for version 11.0

### PR DESCRIPTION
**Pull request by REALMWTH**
# SCP:SL Translation PR Template

## Language Name
Russian

## State Type of change (New language, changed existing Etc.)
Updated Russian localization for 11.0 patch
dado#9223 helped me to make this update ;)

# Checklist:
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

**Comment by REALMWTH** [comment]
There are some of the issues within the game that prevents correct quality translation:
https://github.com/northwood-studios/SCPSL-Translations/issues/778
https://github.com/northwood-studios/SCPSL-Translations/issues/779

File: Translations/ru/Class_Names.txt [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+Мобильная Опергруппа — Капитан
+Мобильная Опергруппа — Рядовой
+Обучение
 Охранник Комплекса
 SCP-939-53
-SCP-939-89
\ No newline at end of file
+SCP-939-89
+Повстанцев Хаоса — Стрелок
+Повстанцев Хаоса — Репрессор
```
Comment (GuyFawkes76) [inline Translations/ru/Class_Names.txt:20]: Усмиритель? Но, в принципе, можно оставить и так.

File: Translations/ru/Class_Names.txt [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+Персонал Класса D
 Наблюдатель
 SCP-106
-Мобильная Опергруппа — Научный Сотрудник
+Мобильная Опергруппа — Специалист
 SCP-049
 Научный Сотрудник
 SCP-079
-Повстанец Хаоса
+Повстанцев Хаоса — Последователь
```
Comment (GuyFawkes76) [inline Translations/ru/Class_Names.txt:9]: Не думаю, что Последователь - правильное слово здесь для обозначения слова Conscript

File: Translations/ru/NewMainMenu.txt [hunk 3]
```diff
@@ -57,7 +57,7 @@ SCP-079 - Пост-процессинг
 Главное Меню
 Подключиться
 Назад
-Эти изменения требуют перезапуск.\nПерезагрузить меню сейчас?
+Эти изменения требуют перезапуск меню.\nПерезагрузить меню сейчас?
```
Comment (GuyFawkes76) [inline Translations/ru/NewMainMenu.txt:60]: Возможно, "перезапуска"?

**Review by GuyFawkes76 — CHANGES_REQUESTED** [review]
Кроме того, что указал отдельными комментариями, ещё есть сомнения в падежах в файле Categories и в случае Повстанцев Хаоса. (У меня сейчас нет возможности проверить в игре, поэтому просто проверьте ещё раз, что всё ок)

File: Translations/ru/NewMainMenu.txt (hunk 3 above)
Comment (REALMWTH) [reply Translations/ru/NewMainMenu.txt:60]: Исправлено

File: Translations/ru/Class_Names.txt (hunk 2 above)
Comment (REALMWTH) [reply Translations/ru/Class_Names.txt:9]: Заменил на Новобранец

File: Translations/ru/Class_Names.txt (hunk 1 above)
Comment (REALMWTH) [reply Translations/ru/Class_Names.txt:20]: Оставил как есть

**Comment by REALMWTH** [comment]
@GuyFawkes76 по падежам всё просто. Формат сообщения в hint'e:
Достигнут лимит <color=yellow>[type]</color>

Пример: Достигнут лимит Раций

**Review by GuyFawkes76 — APPROVED** [review]

**Pull request merged**